setup.py

```diff
@@ -1,14 +1,14 @@
 from setuptools import setup, find_packages
 
 setup(name='pypif-sdk',
-      version='2.2.0',
+      version='2.2.1',
       url='http://github.com/CitrineInformatics/pypif-sdk',
       description='High level support for working with Physical Information Files (PIF)',
       author='Max Hutchinson',
       author_email='maxhutch@citrine.io',
       packages=find_packages(),
       install_requires=[
           'pypif>=2.0.0,<4',
-          'citrination_client>=3,<4',
+          'citrination_client>=3,<5',
```

Comment:
This will work with _either_ 3 or 4?

Reply:
I feel that we should probably put the lower bound here at 4, although I admit I don't know the extent of this library's dependency on PyCC.